### PR TITLE
D8CORE-7435 D8CORE-7436 | configure card and list display

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- D8CORE-7435 | configure card display(s)
- D8CORE-7436 | configure list display(s)

# Review By (Date)
- When possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Create a page with an opportunities card
4. Verify that the layout structure matches the [figma wireframe screenshot in JIRA](https://stanfordits.atlassian.net/browse/D8CORE-7435)
5. Create a page with a list containing opportunities 
6. Verify that the layout structure matches what is outlined in [JIRA](https://stanfordits.atlassian.net/browse/D8CORE-7436)
7. Review code

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? **Yes**

# Associated Issues and/or People
- D8CORE-7435
- D8CORE-7435

### CODE REVIEW

File: config/sync/core.entity_view_display.node.stanford_opportunity.teaser.yml [hunk 1]
```diff
@@ -28,30 +28,59 @@ dependencies:
...
     - text
     - user
 id: node.stanford_opportunity.teaser
 targetEntityType: node
 bundle: stanford_opportunity
 mode: teaser
 content:
-  body:
-    type: text_summary_or_trimmed
+  su_opp_application_deadline:
```
Comment (pookmish) [inline config/sync/core.entity_view_display.node.stanford_opportunity.teaser.yml:39]: Did you want to configure the teaser display which will be the "List" display?

File: config/sync/core.entity_view_display.node.stanford_opportunity.stanford_h3_card.yml [hunk 2]
```diff
@@ -102,6 +106,14 @@ content:
         label_tag: ''
     weight: 0
     region: card_image
+  su_opp_sponsor:
+    type: entity_reference_label
+    label: hidden
+    settings:
+      link: true
```
Comment (pookmish) [inline config/sync/core.entity_view_display.node.stanford_opportunity.stanford_h3_card.yml:113]: We don't want to link to the sponsor taxonomy term.

File: config/sync/core.entity_view_display.node.stanford_opportunity.stanford_h3_card.yml [hunk 3]
```diff
@@ -131,7 +143,15 @@ content:
...
         label_value: ''
         label_tag: ''
-    weight: 2
+    weight: 4
+    region: card_body
+  su_opp_type:
+    type: entity_reference_label
+    label: hidden
+    settings:
+      link: true
```
Comment (pookmish) [inline config/sync/core.entity_view_display.node.stanford_opportunity.stanford_h3_card.yml:152]: I don't believe wet want to link to the taxonomy term.

File: config/sync/core.entity_view_display.node.stanford_opportunity.stanford_card.yml [hunk 4]
```diff
@@ -132,7 +143,15 @@ content:
...
         label_value: ''
         label_tag: ''
-    weight: 2
+    weight: 4
+    region: card_body
+  su_opp_type:
+    type: entity_reference_label
+    label: hidden
+    settings:
+      link: true
```
Comment (pookmish) [inline config/sync/core.entity_view_display.node.stanford_opportunity.stanford_card.yml:152]: I don't believe wet want to link to the taxonomy term.

File: config/sync/core.entity_view_display.node.stanford_opportunity.stanford_card.yml [hunk 5]
```diff
@@ -103,6 +106,14 @@ content:
         label_tag: ''
     weight: 0
     region: card_image
+  su_opp_sponsor:
+    type: entity_reference_label
+    label: hidden
+    settings:
+      link: true
```
Comment (pookmish) [inline config/sync/core.entity_view_display.node.stanford_opportunity.stanford_card.yml:113]: We don't want to link to the sponsor taxonomy term.

**Review by pookmish — CHANGES_REQUESTED** [review]
This PR is close, but the base branch needs some work. You can merge them all into one if that makes it easier for you to iterate on.

**Comment by rebeccahongsf** [comment]
> This PR is close, but the base branch needs some work. You can merge them all into one if that makes it easier for you to iterate on.

I'll do just that and merge this PR in. Changes to be implemented in #813 😉 Thank you for the speedy review, Mike!

**Pull request merged**